provisioning/roles/shorewall-firewall/files/vpn-shorewall/shorewall.conf

```diff
@@ -227,10 +298,3 @@ PROVIDER_OFFSET=
 MASK_BITS=
 
 ZONE_BITS=0
-
```

Comment:
DO NOT DELETE

`shorewall update -A` removed it.

Should I put this back in??